utils/helper.sh

```diff
@@ -839,16 +839,8 @@ function create_connect_topics_and_acls() {
   ccloud kafka acl create --allow --service-account $serviceAccount --operation READ --consumer-group connect-cloud
 
   echo "Connectors: creating topics and ACLs for service account $serviceAccount"
-  for topic in __consumer_timestamps ; do
-    ccloud kafka topic create $topic &>/dev/null
-    ccloud kafka acl create --allow --service-account $serviceAccount --operation WRITE --topic $topic --prefix
-    ccloud kafka acl create --allow --service-account $serviceAccount --operation READ --topic $topic --prefix
-  done
   ccloud kafka acl create --allow --service-account $serviceAccount --operation READ --consumer-group connect-replicator
   ccloud kafka acl create --allow --service-account $serviceAccount --operation describe --cluster-scope
-  ccloud kafka acl create --allow --service-account $serviceAccount --operation describe --topic pageviews
-  ccloud kafka acl create --allow --service-account $serviceAccount --operation describe-configs --topic pageviews
-  ccloud kafka acl create --allow --service-account $serviceAccount --operation describe --topic __consumer_timestamps
```

Comment:
+1 for moving the demo specific topic configurations out of the shared function into the demo itself.  I'm curious why you're also removing the describe acl on __consumer_timestamps.  Has something changed to make that no longer necessary?

Reply:
`__consumer_timestamps` was never required in the first place.  This topic is created on the origin cluster, not destination cluster.